cvat/apps/engine/tests/test_rq.py

```diff
@@ -0,0 +1,130 @@
+from datetime import datetime, timedelta
+import uuid
+from unittest import mock
+import unittest
+
+from rq.job import Job, Dependency
+from rq.registry import DeferredJobRegistry, StartedJobRegistry
+import django_rq
+from django_rq.queues import DjangoRQ
+from django.test import override_settings
+from django.conf import settings
+
+from cvat.apps.engine.rq import define_dependent_job, is_rq_job_owner
+from cvat.apps.engine.tests.utils import clear_rq_jobs
+
+
+DEFAULT_USER_ID = 1
+
+
+def dummy_task():
+    return 2 + 2
+
+
+def _create_test_job(
```

Comment:
Probably need to add a comment that all jobs created in these tests will always be queued.

Reply:
Renamed a function. Now it's clear that it adds a job to the queue